src/copy.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
```

Comment:
Instead of disabling 'no-console' rule for the entire file, consider using a dedicated logger or using console only in catch block.